yjit/src/stats.rs

```diff
@@ -283,6 +283,8 @@ make_counters! {
     setivar_frozen,
     setivar_megamorphic,
 
+    ivar_last_chain_count,
```

Comment:
It would make it part of "getinstancevariable exit reasons", so I deliberately avoided that. I'll change it to `get_ivar_last_chain_count` as another workaround 🤔

Reply:
"max_depth" feels like the count itself is a depth, but it's not. Thinking about another name.

Reply:
A bit long, but maybe `get_ivar_max_depth_count`?

Reply:
IMO the current name is fine, but if you prefer the longer version go ahead 😅

Reply:
OK I'll leave the current one then. I'll change it when I get confused myself 🙂